packages.yml

```diff
@@ -2,9 +2,9 @@ packages:
   # - package: fivetran/quickbooks_source
   #   version: [">=0.6.0", "<0.7.0"]
 
-  - git: https://github.com/fivetran/dbt_quickbooks_source.git
-    revision: feature/add-class-ids-to-source-models
-    warn-unpinned: false 
+- git: https://github.com/fivetran/dbt_quickbooks_source.git
+  revision: feature/quickbooks-updates-project
```

Comment:
With `packages.yml` I should keep it the way it is for now since it's dependent on `features/quickbook-updates-project` for the time being, but will change when the branch is ready to merge.